tools/mob_suite/mob_typer.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="mob_typer" name="MOB-Typer" version="1.4.1">
+  <description>(Get the plasmid type and mobility given its sequence)</description>
+  <requirements>
+     <requirement type="package" version="1.4.1">mob_suite</requirement>
+     <!--requirement type="package" version="2.7.1">blast</requirement-->
+     <!--requirement type="package" version="2.0">mash</requirement-->
+     <!--requirement type="package" version="1.5.5">circlator</requirement-->
+  </requirements>   
+  <command detect_errors="exit_code">
+    ln -s ${input} ${input.name};
```

Comment:
Be careful with `${input}`, you need to add `'${input}'` to take the input name as literal in case the user enters something weird.